### ISSUE TITLE
Missing loading of pdf.js (& other required dependencies) in apps with custom deploy URL

### ISSUE THREAD

**Issue by sniperwolf** [stephanrauh/ngx-extended-pdf-viewer#284]
When an angular app deploy URL is different from `/` (ex. `/path`), `pdf.js` and all the other required dependencies loaded by `ngx-extended-pdf-viewer` are not reachable (`404`).

**Version info**

- Version of ngx-extended-pdf-viewer is **3.2.3**
- pdf.js, pdf-worker.js and viewer.js are the default files

**To Reproduce**

I reproduce the bug [here](https://github.com/sniperwolf/ngx-extended-pdf-viewer-issue-284).

```shell script
git clone https://github.com/sniperwolf/ngx-extended-pdf-viewer-issue-284.git
cd ngx-extended-pdf-viewer-issue-284/
npm install
npm run start
```

It should open a new browser tab with `http://localhost:4200/path` and the app component with `ngx-extended-pdf-viewer` and the `404` error in the console (the browser try to load `http://localhost:4200/assets/pdf.js` but the correct URL is `http://localhost:4200/path/assets/pdf.js`).

I just change the app `deployUrl` in the `package.json` to serve it in `localhost:4200/path`.

**Additional context**

The component creates the new script element to load `pdf.js` with the hardcoded `/asset/` in `src` attribute. Check [here](https://github.com/stephanrauh/ngx-extended-pdf-viewer/blob/master/projects/ngx-extended-pdf-viewer/src/lib/ngx-extended-pdf-viewer.component.ts#L504).

[Here](https://github.com/stephanrauh/ngx-extended-pdf-viewer/blob/master/projects/ngx-extended-pdf-viewer/src/lib/ngx-extended-pdf-viewer.component.ts#L522) there is the same behavior for the `viewer.js` dependency.

**Comment by Gr33nbl00d** [comment]
Same problem here...

**Comment by stephanrauh** [comment]
The bugfix has landed with version 3.2.4.

Thanks a lot for the reproducer and the detailed analysis. It helped me a lot!

Best regards,
Stephan
